desktop/cmp/input/Checkbox.ts

```diff
@@ -71,8 +71,8 @@ const cmp = hoistCmp.factory<CheckboxInputModel>(({model, className, ...props},
         inline: withDefault(props.inline, true),
         label: props.label,
         tabIndex: props.tabIndex,
-
         id: props.id,
+        'data-testid': props.testId,
```

Comment:
We should add `testId` to `HoistInputProps` - I think our goal (hopefully we achieve it) is to support testId for all inputs, and adding it there means it's officially declared for every one of them.